locales/en/messages.json

```diff
@@ -4875,7 +4875,14 @@
     "osdDescElementEfficiency": {
         "message": "Instantaneous battery consumption in mAh/distance.  (Requires valid GPS fix)"
     },
-    
+    "osdTextTotalFlights": {
+        "message": "Total flights",
+        "description": "One of the elements of the OSD"
+    },
+    "osdDescTotalFlights": {
+        "message": "Estimated total flights"
```

Comment:
Not sure 'estimated' is the right word here - we actually count them and do not use some estimation technique. Would 'approximate' be better, to account for the fact that this is not necessarily foolproof? Probably also better make it 'Approximate total number of flights'.